pilot/pkg/model/network.go

```diff
@@ -482,11 +491,19 @@ func (n *networkGatewayNameCache) resolveAndCache(name string) []string {
 		ttl = MinGatewayTTL
 	}
 	expiry := time.Now().Add(ttl)
+	refreshPeriod := ttl
+	if len(addrs) == 0 {
+		// resolution failed, retry after 5s (by default)
```

Comment:
every 5s forever may be high load. May want backoff? 

30s seems reasonable  for failure if no backoff?

Reply:
I just use the default DNS refresh rate in Envoy

During the downtime we just send DNS requests every 5s without pushing xDS, so I think it's fine 
IMO 30s or backoff may be a bit too long for the system to recover

Reply:
This is 17280 requests per day per hostname per istiod, if its constantly in a failure state, can be very high cost

Reply:
Got it, changed to 30s